src/client/setup.py

```diff
@@ -17,10 +21,10 @@
 def load_conf():
     """Load the build config file"""
     file_self = os.path.dirname(os.path.abspath(__file__))
-    while file_self != '/':
-        new_file = os.path.join(file_self, '.build_vars.json')
+    while file_self != "/":
```

Comment:
Why change single quotes to double quotes? We aren't particular about it, but single quotes are at least easier to type.

Reply:
Good question.  I ran this whole file though black to try and get consistency but I can revert the quotes change if you prefer.  It seems the python preferred is double but I also prefer single personally.

Reply:
I'm ambivalent. We don't have a formal standard anyway